examples/sensor_connectivity.py

```diff
@@ -52,24 +55,18 @@
     baseline=(None, 0),
     reject=dict(grad=4000e-13, eog=150e-6),
 )
+epochs.load_data().pick("grad")  # just keep MEG and no EOG now
 
-# Compute connectivity for band containing the evoked response.
-# We exclude the baseline period:
-fmin, fmax = 4.0, 9.0
-sfreq = raw.info["sfreq"]  # the sampling frequency
+# Compute Fourier coefficients for the epochs (returns an EpochsSpectrum object)
+# (storing Fourier coefficients in EpochsSpectrum objects requires MNE >= 1.8)
 tmin = 0.0  # exclude the baseline period
-epochs.load_data().pick_types(meg="grad")  # just keep MEG and no EOG now
+spectrum = epochs.compute_psd(method="multitaper", tmin=tmin, output="complex")
+
+# Compute connectivity for the frequency band containing the evoked response
+# (passing EpochsSpectrum objects as data requires MNE-Connectivity >= 0.8)
+fmin, fmax = 4.0, 9.0
 con = spectral_connectivity_epochs(
-    epochs,
-    method="pli",
-    mode="multitaper",
-    sfreq=sfreq,
-    fmin=fmin,
-    fmax=fmax,
-    faverage=True,
-    tmin=tmin,
-    mt_adaptive=False,
-    n_jobs=1,
+    data=spectrum, method="pli", fmin=fmin, fmax=fmax, faverage=True, n_jobs=1
 )
```

Comment:
I checked by eye this example and it did indeed look the same which is good!

https://output.circle-artifacts.com/output/job/af5d79d3-61cd-4dd6-af69-a46142b3ce5d/artifacts/0/dev/auto_examples/sensor_connectivity.html#sphx-glr-auto-examples-sensor-connectivity-py

Reply:
Yeah, everything I tried with multitaper was identical, just some minor deviations for the Fourier/Welch.